BitwardenShared/UI/Platform/Settings/Settings/AutoFill/PasswordAutoFill/PasswordAutoFillView.swift

```diff
@@ -5,9 +5,97 @@ import SwiftUI
 /// A view that shows the instructions for enabling password autofill.
 ///
 struct PasswordAutoFillView: View {
+    // MARK: Properties
+
+    /// The store used to render the view.
+    @ObservedObject var store: Store<
+        PasswordAutoFillState,
+        Void,
+        PasswordAutoFillEffect
+    >
+
+    /// An object used to determine the current color scheme.
+    @Environment(\.colorScheme) var colorScheme
+
+    /// An object used to open urls from this view.
+    @Environment(\.openURL) private var openURL
+
     // MARK: View
 
     var body: some View {
+        Group {
+            if store.state.nativeCreateAccountFeatureFlag {
+                contentView
+            } else {
+                legacyContentView
+            }
+        }
+        .navigationBar(
+            title: store.state.navigationBarTitle,
+            titleDisplayMode: .inline
+        )
+        .task {
+            await store.perform(.appeared)
+        }
+    }
+
+    // MARK: Private Views
+
+    /// The content view.
+    private var contentView: some View {
+        VStack(spacing: 0) {
+            ZStack {
+                gifViewPlaceholder
+                    .frame(width: 230, height: 278)
+                    .background(.red)
+                    .padding(.top, 32)
+
+                gifView
+                    .frame(width: 230, height: 280)
+                    .padding(.top, 32)
+            }
```

Comment:
🤔 Does the gif gets displayed correctly on a really small device like iPhone 6s or iPhone SE? What about landscape mode?

Reply:
yeah it does!  I am going to do some work here on this [ticket](https://bitwarden.atlassian.net/browse/PM-10803) to have it match the figma designs so I can make sure everything is good on that PR if thats cool with you?